controllers/mobile/frmPresentersController/frmPresentersController.js

```diff
@@ -48,24 +33,7 @@ define({
     };
   },
 
-    /**
-     * @function  setFilteronClick
-     * @description This function is used to set Filter Click Action
-     * @private
-     */
-    setFilteronClick: function() {
-        var self = this;
-        this.view.filterAll.onClick = function(eventobject) {
-            self.speakerFilter(eventobject);
-        };
-        this.view.filterDBX.onClick = function(eventobject) {
-            self.speakerFilter(eventobject);
-        };
-        this.view.filterQuantum.onClick = function(eventobject) {
-            self.speakerFilter(eventobject);
-        };
-    },
-
+  
```

Comment:
Please remove the below unused functions 
1.presenterFetchSuccess
2.presenterFetchFailure